data/json/recipes/electronics/tools.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "result": "makeshift_oscilloscope",
+    "type" : "recipe",
+    "category": "CSC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 2 ],
+    "time": 240000,
+    "book_learn": [ [ "advanced_electronics", 4 ], [ "mag_electronics", 6 ] , [ "manual_electronics", 5 ] ],
+    "qualities": [
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "MULTIMETER", "level": 2 }
+    ],
+    "tools": [ [ [ "soldering_iron", 20 ], [ "toolset", 20 ] ] ],
+    "components": [
+      [ [ "television", 1 ] ],
```

Comment:
You mean TV or LCD? Adding TV as an alternative part would mean uncraft recipe would be quite weird (craft with TV, uncraft into ram and processors?) and would also prevent the recipe for being properly reversible (reversing an item made with TV would drop a TV).

LCD needs more uses anyway. It would be better to make it with LCD by default.